File: PharmaDicBackEnd/PharmaDicBackEnd.ApiService/Models/Symptom.cs
using System;
using System.Collections.Generic;

namespace PharmaDicBackEnd.ApiService.Models;

public partial class Symptom
{
    public int SymptomId { get; set; }

    public string SymptomName { get; set; } = null!;

    public string? Description { get; set; }

    public virtual ICollection<Disease> Diseases { get; set; } = new List<Disease>();
}

File: PharmaDicMobile/app/src/main/java/com/example/pharmadicmobile/models/Symptom.java
package com.example.pharmadicmobile.models;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class Symptom implements Serializable {
    private int symptomId;
    private String symptomName;
    private String description;
    private List<Disease> diseases = new ArrayList<>();

    public Symptom() {}

    public int getSymptomId() { return symptomId; }
    public void setSymptomId(int symptomId) { this.symptomId = symptomId; }

    public String getSymptomName() { return symptomName; }
    public void setSymptomName(String symptomName) { this.symptomName = symptomName; }

    public String getDescription() { return description; }
    public void setDescription(String description) { this.description = description; }

    public List<Disease> getDiseases() { return diseases; }
    public void setDiseases(List<Disease> diseases) { this.diseases = diseases; }
}
